scripts/lib/mlkem_test.py

```diff
@@ -260,6 +260,25 @@ def run_schemes(
         else:
             return results
 
+    def test(
+        self,
+        opt: bool,
+        compile: bool,
+        run: bool,
+        extra_make_envs={},
+        extra_make_args=[],
+        actual_proc: Callable[[bytes], str] = None,
+        expect_proc: Callable[[SCHEME, str], tuple[bool, str]] = None,
```

Comment:
Can you document, here and in the other functions sharing this part of the signature, what the arguments mean? Esp. `actual_proc` and `expected_proc` are not self-explanatory.